### PR TITLE
Kibana 8.15.0 new node path support

### PR DESCRIPTION
**Pull request by Dzuming**
- Kibana 8.15.0 new node path support

### CODE REVIEW

File: ror-demo-cluster/images/kbn/install-ror-kbn-using-api.sh [hunk 1]
```diff
@@ -25,6 +29,11 @@ echo "Installing KBN ROR $ROR_VERSION..."
 /usr/share/kibana/bin/kibana-plugin install "https://api.beshu.tech/download/kbn?esVersion=$KBN_VERSION&pluginVersion=$ROR_VERSION&edition=$ROR_KBN_EDITION&email=ror-sandbox%40readonlyrest.com"
 if verlte "7.9.0" "$KBN_VERSION"; then
```
Comment (coutoPL) [inline ror-demo-cluster/images/kbn/install-ror-kbn-using-api.sh:30]: WDYT about doing it like this?
```
if vergte "8.15.0" "$KBN_VERSION"; then
  echo "Patching KBN ROR $ROR_VERSION..."
  /usr/share/kibana/node/glibc-217/bin/node plugins/readonlyrestkbn/ror-tools.js patch;
elif verlte "7.9.0" "$KBN_VERSION"; then
  echo "Patching KBN ROR $ROR_VERSION..."
  /usr/share/kibana/node/bin/node plugins/readonlyrestkbn/ror-tools.js patch;
fi
```
we obviously have to duplicate `echo`, but IMO it's more readable

File: ror-demo-cluster/images/kbn/install-ror-kbn-using-file.sh [hunk 2]
```diff
@@ -13,6 +17,11 @@ echo "Installing KBN ROR from file..."
 /usr/share/kibana/bin/kibana-plugin install file:///tmp/ror.zip
 if verlte "7.9.0" "$KBN_VERSION"; then
```
Comment (coutoPL) [inline ror-demo-cluster/images/kbn/install-ror-kbn-using-file.sh:18]: the same here

**Review by coutoPL — APPROVED** [review]
LGTM

File: ror-demo-cluster/images/kbn/install-ror-kbn-using-api.sh [hunk 3]
```diff
@@ -25,6 +29,11 @@ echo "Installing KBN ROR $ROR_VERSION..."
...
   echo "Patching KBN ROR $ROR_VERSION..."
-  /usr/share/kibana/node/bin/node plugins/readonlyrestkbn/ror-tools.js patch
+
+  if vergte "8.15.0" "$KBN_VERSION"; then
+    /usr/share/kibana/node/glibc-217/bin/node plugins/readonlyrestkbn/ror-tools.js patch;
```
[the remark this answers is not in the document]
Comment (Dzuming) [reply ror-demo-cluster/images/kbn/install-ror-kbn-using-api.sh]: Yes, good point. I didn't have time to do it yet, but the node path is different based on the Kibana installation way, so I need to verify it.

[the remark this answers is not in the document]
Comment (coutoPL) [reply ror-demo-cluster/images/kbn/install-ror-kbn-using-api.sh]: ok, I'm resolving this comment, but please don't forget about the topic. 
Let's merge this one

[the remark this answers is not in the document]
Comment (Dzuming) [reply ror-demo-cluster/images/kbn/install-ror-kbn-using-api.sh]: mr https://github.com/beshu-tech/readonlyrest-docs/pull/217

**Pull request merged**